sdk/template/azure_template_core/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "azure_template_core"
+version = "0.1.0"
+description = "Template crate for Azure SDK pipeline testing"
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+
+[dependencies]
+
+[dev-dependencies]
+azure_template.path = "../azure_template"
```

Comment:
`azure_template` should depend on `azure_template_core`, actually, to fit with our convention.

Reply:
It does, but we also need to model the circular dev-dependency

Reply:
Ah, gotcha. I missed that.